### DISCUSSION TITLE
Color de los horarios con base en cupos disponibles

### DISCUSSION THREAD

**Discussion by c4ts0up** [Open-Source-Uniandes/Mi-Horario-Uniandes#44] [Ideas]
Con el fin de mejorar la visualización de los horarios creados, los recuadros correspondientes a las secciones podrían rellenarse con colores que representen la disponibilidad de la clase. La disponibilidad se entendería como el número de cupos disponibles sobre el número de cupos totales. De este modo, digamos, se podría hacer lo siguiente:

Si la disponibilidad es:
- \>= 80% : verde oscuro
- \>= 60% : verde claro
- \>= 40% : amarillo
- \>= 20 % : naranja
- \>= 0% : rojo
- 0 % : negro

**Comment by jsurrea** [comment]
Me gusta la idea, hagámoslo!

**Comment by jsurrea** [comment]
La única consideración que le veo es ¿Cómo mostrar cuando una sección ha sido seleccionada? ¿Usamos otro color? En este momento se usa el verde

**Reply by c4ts0up** [reply]
Podríamos usar un bordeado grueso con un color vibrante. O podríamos utilizar tonos menos brillantes para mostrar que no ha sido seleccionado y cuándo ha sido seleccionado, usar tonos vibrantes
